frontend/packages/knative-plugin/src/components/overview/KSRoutesOverviewListItem.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { K8sResourceKind, referenceForModel } from '@console/internal/module/k8s';
+import { ResourceLink, ExternalLink } from '@console/internal/components/utils';
+import { RouteModel } from '../../models';
+
+type KSRoutesOverviewListItemProps = {
+  ksroute: K8sResourceKind;
+};
+
+const KSRoutesOverviewListItem: React.FC<KSRoutesOverviewListItemProps> = ({ ksroute }) => {
+  const {
+    metadata: { name, namespace },
+    status,
+  } = ksroute;
+  return (
+    status && (
+      <li className="list-group-item">
+        <div className="row">
+          <div className="col-xs-10">
+            <ResourceLink kind={referenceForModel(RouteModel)} name={name} namespace={namespace} />
+            {status?.url?.length > 0 && (
```

Comment:
nit: on line 16 you checked for status so may not need here 
```suggestion
            {status.url?.length > 0 && (
```

Reply:
> Should be consistent IMO for both revision and ksvc sidebar,
> 
>     1. Either show base route always in location and then traffic percent(split) with route
> 
>     2. Or in case of just base route and it receiving 100% traffic , show Percentage as well in route block itself as shown for revision
> 
> 
> @beaumorley @serenamarie125 any thoughts/comments?

I think the second method of displaying the route once might be better, since it wouldn't be useful to show the same URL twice.

I've updated the `KSVC` sidebar to match the route section for `Revision`:
![img](https://user-images.githubusercontent.com/20013884/88642418-1614cb00-d0de-11ea-82ee-7333b62e41ea.png)

Reply:
Removed.